### PR TITLE
Remove Authorization Headers From Debug Response

### PR DESCRIPTION
**Pull request by SyntaxNode**
Fixes https://github.com/prebid/prebid-server/issues/1776

### CODE REVIEW

**Review by hhhjort — APPROVED** [review]
LGTM

File: exchange/bidder.go [hunk 1]
```diff
@@ -327,23 +327,39 @@ func getAssetByID(id int64, assets []nativeRequests.Asset) (nativeRequests.Asset
...
 		}
 	}
+
+	return &openrtb_ext.ExtHttpCall{}
+}
+
+var authorizationHeader = http.CanonicalHeaderKey("authorization")
+
+func filterHeader(h http.Header) map[string][]string {
+	clone := h.Clone()
```
Comment (mansinahar) [inline exchange/bidder.go:360]: Wondering if we need to clone the http header map here? Do we expect the authorization header to be used anywhere else outside of this function? If so then it makes sense to not modify the original object but if not then we can just delete it from the original object

File: exchange/bidder.go [hunk 2]
```diff
@@ -327,23 +327,39 @@ func getAssetByID(id int64, assets []nativeRequests.Asset) (nativeRequests.Asset
...
+				RequestBody:    string(httpInfo.request.Body),
+				RequestHeaders: filterHeader(httpInfo.request.Headers),
+			}
 		}
 	}
+
+	return &openrtb_ext.ExtHttpCall{}
+}
+
+var authorizationHeader = http.CanonicalHeaderKey("authorization")
```
Comment (mansinahar) [inline exchange/bidder.go:357]: We should make this a constant variable as we don't expect any code to change it

File: exchange/bidder.go (hunk 1 above)
Comment (mansinahar) [reply exchange/bidder.go:360]: I checked the code and I don't see any other usage of the Authorization header so it should be safe to remove it from the original object here

File: exchange/bidder.go (hunk 2 above)
Comment (SyntaxNode) [reply exchange/bidder.go:357]: In Go, constants must be a hardcoded value. The function call to `CanonicalHeaderKey` prevents me from marking it as a constant.

**Review by mansinahar — APPROVED** [review]

**Review by hhhjort — APPROVED** [review]

**Pull request merged**